docs/en/FAQ.md

```diff
@@ -1,104 +1,98 @@
 ---
 layout: global
-title: FAQ
+title: Frequently Asked Questions
 nickname: FAQ
 group: Home
 priority: 0
 ---
 
-## Alluxio Frequently Asked Questions.
+* Table of Contents
+{:toc}
 
-1. What is Alluxio?
+## What is Alluxio?
 
-   [Alluxio](http://www.alluxio.org/), formerly Tachyon, is an open source, memory speed, virtual
-   distributed storage. It enables any application to interact with any data from any storage
-   system at memory speed.
+[Alluxio](http://www.alluxio.org/), formerly Tachyon, is an open source, memory speed, virtual
+distributed storage. It enables any application to interact with any data from any storage system
+ at memory speed. Read more about Alluxio [Overview]({{ site.baseurl }}{% link en/Overview.md %}).
 
-2. What platforms and Java versions does Alluxio run on?
+## What platforms and Java versions can Alluxio run on?
 
-   Alluxio can run with Java 1.8 or above on various distributions of Linux / Mac.
+Alluxio requires JVM 1.8 or above to run on various distributions of Linux / MacOS.
 
-3. Is there an easy way to see the status and health of a cluster?
+## What license is Alluxio under?
 
-   Yes, Once Alluxio is installed, the UI is available at `http://master_ip:19999`
+Alluxio is open sourced under the Apache 2.0 license.
 
-4. If I add new nodes to the cluster, will Alluxio re-balance / move the cached blocks to the newly
-added nodes in order to balance memory space utilization between the nodes?
+## My analytics job does not get faster after deploying Alluxio, why?
 
-   No, Currently re-balancing of data blocks in not available.
+There are multiple possible reasons to check out:
+1. The job can be computation bound and does not spend too much time in reading/writing data.
+In this case, speeding up I/O performance will not help too much.
+1. The persistent storage is colocated with compute (e.g., Alluxio is connected to a local
+HDFS), and the input data of the job is buffer-cached.
```

Comment:
a lot of users might not be familiar with buffer cache, could be easier to understand if we put `... is in the OS [buffer cache](https://www.tldp.org/LDP/sag/html/buffer-cache.html)`

Reply:
added, thanks